ios/BrightcovePlayer.m

```diff
@@ -319,4 +330,32 @@ -(void)playerView:(BCOVPUIPlayerView *)playerView didTransitionToScreenMode:(BCO
     }
 }
 
+- (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(NSDictionary *)change context:(void *)context {
+    
+    if ([keyPath isEqualToString:@"timedMetadata"])
+    {
+        AVPlayerItem* playerItem = object;
+        
+        for (AVMetadataItem* metadata in playerItem.timedMetadata)
+        {
+            
+            if([metadata.key isEqual:@"CREW"]){
```

Comment:
Who sets metadata keys to be exactly CREW? I assume this is a filter of sorts, so we aren't emitting events on metadata that we don't define. But using a string feels a little brittle. If this is something a non-engineer video manager would have to get right, it would make sense to do a toLowerCase (equiv in Objective-C) before comparing. If it's something you're coding up in the producer, it would be good to have a comment on either side referencing the path to the files where "CREW" is hard-coded, so it's explicit what needs to match. Thx. :-)

Reply:
It'll be setup in a script.  The convention for ID3 seems to be uppercase.  I will add the documentation and reference, that's a great idea